src/Surfnet/Stepup/Identity/Event/SecondFactorVettedEvent.php

```diff
@@ -32,13 +32,27 @@
 use Surfnet\Stepup\Identity\Value\VettingType;
 use Surfnet\StepupBundle\Value\SecondFactorType;
 use Surfnet\StepupMiddleware\CommandHandlingBundle\SensitiveData\Forgettable;
+use Surfnet\StepupMiddleware\CommandHandlingBundle\SensitiveData\RightToObtainDataInterface;
 use Surfnet\StepupMiddleware\CommandHandlingBundle\SensitiveData\SensitiveData;
 
 /**
  * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
  */
-class SecondFactorVettedEvent extends IdentityEvent implements Forgettable
+class SecondFactorVettedEvent extends IdentityEvent implements Forgettable, RightToObtainDataInterface
 {
+    protected static $whitelist = [
+        'identity_id',
+        'name_id',
+        'identity_institution',
+        'second_factor_id',
+        'second_factor_type',
+        'preferred_locale',
+        'email',
+        'common_name',
+        'second_factor_identifier',
+        'vettingType',
```

Comment:
This seems to be incorrect. Looking at the Sensitive Data entity, this should be: `vetting_type`